### ISSUE TITLE
Feature request: Contextual reply

### ISSUE THREAD

**Issue by webratte** [open-xchange/ox-coi#111]
Maybe it could work this way:

OX-Coi hide quoted text of regular mails and only shows "..."
So it can detected (maybe by the “>”).

So now you could show this text in the top of the message simmilar how it's done by every famous messenger.

If a OC-Coi user tab the butten “quote” it could added the qouted message behind the regular text as in regular mail.

Example for the sendet message shown in a regular mail client:

#######################

This is the reply of the quoted message

Sendet by xyz@vbh.com
 >This is the message which is quoted.
 >You know what I mean?

######################

And this message should be shown in OX-Coi like this (but nicer 😉):

#######################

     | Name of the sender wrote:
     | This is the message which is quoted.
     | You know what I mean?

This is the reply of the quoted message

######################

**Comment by robert-virkus** [comment]
Hi webratte, many thanks for your suggestion!

I'm not sure if I understand your request correctly. 

- Do you want two reply-options - one without context and one with the context of the last received message?
- Do you want to have an option to view all details of a message aka a "show details" option?
- Or do you want to be able to quote and answer any previous message?


For the latter, we could possibly have this logic:
- when detecting quoted text in a "legacy-reply", i.e. in a message without Chat-Version header, then we would hide quoted text by default
- When, however, a quote exists and this is a chat message, i.e. the Chat-Version header is present, then we could show the quoted text.

Is this what you would like to achieve?

Thanks in advance - Robert

**Comment by webratte** [comment]
Yes, my idea is to be able to quote and answer any previous message.

Similar to other famous messengers.

And I also like your suggestion how to handle it.


Maybe for hided replay text (without chat header), in addition to your suggested solution add a option  to show the hided text.
E.g. long press on this message > show quoted text.

My description was only a idea how to show (the same) quoted message in OX Coi and in regular mail.

I hope you will understand me.
I know my english is really bad 🤔

**Comment by webratte** [comment]
To illustrate my idea I have created two screenshots:

For OX-Coi it should looks like this:

![dc](https://user-images.githubusercontent.com/6864851/66265994-46257180-e81f-11e9-8153-28a3574d3254.png)

And in a normal Mail Client (k9 in my example) it should looks like:

![k9](https://user-images.githubusercontent.com/6864851/66266007-63f2d680-e81f-11e9-9f8c-70da2331896a.png)
